src/object/moving_sprite.cpp

```diff
@@ -120,13 +123,32 @@ MovingSprite::matches_sprite(const std::string& sprite_file) const
   return m_sprite_name == sprite_file || m_sprite_name == "/" + sprite_file;
 }
 
+std::string
```

Comment:
Same here, please use references for strings:
```c++
const std::string&
```

Reply:
Causes problems when used with simplesquirrel.

Reply:
Can you tell me exactly what those problems are?

Reply:
Returning a string reference can somewhat corrupt the string.